<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FORM ALAT</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>341</width>
     <height>511</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="iDALATLabel">
        <property name="text">
         <string>ID ALAT</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtId"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="jENISALATLabel">
        <property name="text">
         <string>JENIS ALAT</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmbJenis">
        <item>
         <property name="text">
          <string>Dipetik</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Digesek</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ditiup</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dipukul</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ditekan</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="tIPEALATLabel">
        <property name="text">
         <string>TIPE ALAT</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmbTipe">
        <item>
         <property name="text">
          <string>Gitar</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ukulele</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Kecapi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Biola</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Selo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rebab</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Seruling</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Terompet</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Klarinet</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gendang</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Drum set</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Keyboard</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mERKLabel">
        <property name="text">
         <string>MERK</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtMerk"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="wARNALabel">
        <property name="text">
         <string>WARNA</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txtWarna"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="sEWAPERHAARILabel">
        <property name="text">
         <string>SEWA PERHARI</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="txtSewa"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnTambah">
         <property name="text">
          <string>TAMBAH</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnUpdate">
         <property name="text">
          <string>UPDATE</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="btnHapus">
         <property name="text">
          <string>HAPUS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
